tools/azsdk-cli/Azure.Sdk.Tools.Cli/Tools/TspTool/TspTool.cs

```diff
@@ -79,6 +97,17 @@ private void HandleConvertCommand(InvocationContext ctx, CancellationToken ct)
             output.Output(result);
         }
 
+        private void HandleInitCommand(InvocationContext ctx, CancellationToken ct)
+        {
+            var outputDirectory = ctx.ParseResult.GetValueForOption(outputDirectoryArg);
+            var template = ctx.ParseResult.GetValueForOption(templateArg);
+            var serviceNamespace = ctx.ParseResult.GetValueForOption(serviceNamespaceArg);
+
+            TspToolResponse result = InitTypeSpecProject(outputDirectory: outputDirectory, template: template, serviceNamespace: serviceNamespace);
```

Comment:
I think it's useful to pass the cancellation token down so it eventually gets used by the process helper and we can ctrl-c.

Reply:
Then if we need to set global timeouts later on it's also supported.

Reply:
@benbp I updated process helper and npx helper to be async so that they accept cancellation tokens now.

Reply:
@chrisradek ah my bad I should have mentioned I have a PR up for process helper async (https://github.com/Azure/azure-sdk-tools/pull/11614) but I still have some work to do. I forgot that it's still sync in main.

Reply:
Oh cool! Is that close to getting merged? I can back this change out or just wait for yours to get merged and resolve conflicts here, depending on timelines.